src/tokens/docs/font.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 font:
-  text:
-    value: "'Roboto', sans-serif"
+  face:
```

Comment:
FYI, I wanted to use `family` here, but that didn't show up at all in the compiled tokens. Since that's only the docs I don't care too much and chose a different key...